py/torch_tensorrt/dynamo/conversion/converter_utils.py

```diff
@@ -194,6 +194,7 @@ def create_constant(
     value: Union[int, float, bool, np.ndarray, torch.Tensor],
     name: str,
     dtype: Optional[Union[torch.dtype, np.dtype, TRTDataType, _enums.dtype]],
+    rank: Optional[int] = 1,
```

Comment:
It seems this is used as a toggle to enable 0D tensors, since it does not seem to have an effect for any value other than `rank=0`. Could it be renamed to `min_rank`, or switched to a boolean such as `empty_tensor: bool = False`?